### PR TITLE
Add about tab to sidebarVC, with info on team, CUAppDev, and links

### PR DESCRIPTION
**Pull request by ChenJesse**
https://github.com/cuappdev/tempo/issues/54

### CODE REVIEW

File: Tempo/Controllers/AboutViewController.swift [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import UIKit
+
+class AboutViewController: UIViewController {
+	@IBOutlet weak var aboutLabel: UILabel!
+	@IBOutlet weak var teamLabel: UILabel!
+    @IBOutlet weak var linksLabel: UITextView!
+	
+    let aboutText = "Cornell App Development is a project team at Cornell University dedicated to mobile app development. Every year, we take an app from idea to product, culminating with a release on the App Store. We give students the opportunity to work in a startup environment and gain practical experience in software development, design, and product management. Our team is made up of talented designers, developers, and visionaries who collaborate to bring projects to life."
+	let teamText = "Annie Cheng, Andrew Aquino, Dennis Fedorko, Ilan Filonenko, Adam ElShaer, Eileen Dai, Austin Chan, Sahil Khoja, Sara Cheong, Tiffany Zheng, Derrick Ho, Jason Wu, Mihir Chauhan, Jonathan Lee, Mark Bryan, Jesse Chen, Joseph Antonakakis, Emily Lien, Alex Zielenski, Natasha Armbrust, Daniel Li, Hong Jeon, Logan Allen, Celine Brass, Kevin Greer, Monica Ong, Amit Mizrahi, Shiv Roychowdhury, Matt Barker, Austin Astorga, Keivan Shahida, Arman Esmaili, Sanjana Kaundinya, Rishab Gupta, Ji Hun Kim"
```
Comment (anniexcheng) [inline Tempo/Controllers/AboutViewController.swift]: Alex isn't on the team anymore. Also, let's divide this into 3 sections (iOS, backend, and design), and order people alphabetically in these these sections.

File: Tempo/Controllers/AboutViewController.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	let linksText = "http://www.cuappdev.org/ \n" + "https://github.com/cuappdev/ \n" + "https://twitter.com/cornellappdev/ \n"
```
Comment (anniexcheng) [inline Tempo/Controllers/AboutViewController.swift]: See if we can add deep linking so that clicking on a link with open up our site.

**Comment by anniexcheng** [comment]
Let's add "About" as the navigation bar title and make the tempo logo a little bigger.

simulator screen shot oct 2 2016 3 33 33 pm

**Comment by ChenJesse** [comment]
Ok I'll comment when I make all the changes

**Comment by ChenJesse** [comment]
Alright, made the changes. The links should have been clickable before the changes, and I added "Leads" to the team because I wasn't sure where to put people.

**Comment by anniexcheng** [comment]
Looks good!

**Comment by loganallen** [comment]
Joe is on backend. Other than that!

**Comment by anniexcheng** [comment]
^ oh right, good catch

**Comment by ChenJesse** [comment]
Ok I'll fix it and then merge it in!

**Pull request merged**